MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffFittingPresenter.h

```diff
@@ -76,6 +76,16 @@ class MANTIDQT_CUSTOMINTERFACES_DLL EnggDiffFittingPresenter
                      std::string tableName, size_t row, std::string &startX,
                      std::string &endX);
 
+  void browsePeaksToFit();
+
+  void addPeakToList();
+
+  void savePeakList();
+
+  std::string readPeaksFile(std::string fileDir);
+
+  void fittingWriteFile(const std::string &fileDir);
+
```

Comment:
These new methods could be moved to the private section of this class. 
That could be done here or later. This also applies to many or most of the methods on the following lines of this file.